.github/workflows/master.yml

```diff
@@ -70,16 +70,18 @@ jobs:
     strategy:
       matrix:
         python-version: [3.6, 3.7]
-        include:
+        spark-version: [2.4.5]
+        pandas-version: [0.24.2, 0.25.3, 1.0.0, 1.0.1]
+        pyarrow-version: [0.13.0, 0.14.1]
+        exclude:
           - python-version: 3.6
-            spark-version: 2.4.5
-            pandas-version: 0.24.2
-            pyarrow-version: 0.13.0
-            logger: databricks.koalas.usage_logging.usage_logger
-          - python-version: 3.7
-            spark-version: 2.4.5
             pandas-version: 0.25.3
+          - python-version: 3.6
             pyarrow-version: 0.14.1
+          - python-version: 3.7
+            pandas-version: 0.24.2
+          - python-version: 3.7
+            pyarrow-version: 0.13.0
```

Comment:
@itholic, I think it's too many. Let's just match with Travis CI's cases for now.

Reply:
ah, i just temporarily added them for checking this PR valid for all version of pandas. i'll restore them after test passed. thanks!! :)